src/main/java/cn/nukkit/Player.java

```diff
@@ -1892,7 +1894,12 @@ protected void processLogin() {
 
         int exp = nbt.getInt("EXP");
         int expLevel = nbt.getInt("expLevel");
-        this.setExperience(exp, expLevel);
+        try {
+            this.setExperience(exp, expLevel);
+        }
+        catch (Exception attributeException){

```

Comment:
Why are we setting the experience to 0 if there is an error? Probably best we fix the issue from the source.

Reply:
I don’t know what causes this, but it just happens sometimes

Reply:
Remove the try catch for now. If you get the error, create an issue and I'll see what I can do about it.